pkg/networkmanager/firewall.jsx

```diff
@@ -878,13 +876,13 @@
                                isChecked={this.state.ipRange == "ip-entire-subnet"}
                                value="ip-entire-subnet"
                                id="ip-entire-subnet"
-                               onChange={(value, e) => this.onChange("ipRange", e.target.value)}
+                               onChange={e => this.onChange("ipRange", e.target.value)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)